packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -2169,6 +2184,88 @@ const TESTS_WITH_PAYPAL_AS_PAYMENT_PROVIDER: [
   ],
 ];
 
+// prettier-ignore
+const TESTS_3MONTHCOUPON: [string, any, Record<string, any>?][] = [
```

Comment:
Instead of creating a new array, could you rather keep using the `TESTS` array, and add the new tests to that? If you need to update any of the `MESSAGE` template values, you can use the `updateTemplateValues` function to do that.

An example of this can be found [here](https://github.com/mozilla/fxa/blob/467c18a100af20ef07b54e44e7eac071a34e2bf7/packages/fxa-auth-server/test/local/senders/emails.ts#L1540)

Reply:
done, thank you!